test/compiler.test.js

```diff
@@ -706,6 +707,47 @@ describe('compiler', function() {
       instructions = boot.compile(appdir.PATH);
       expect(instructions.config).to.not.have.property('modified');
     });
+
+    it('has a mixinSources option', function() {
+      var options = { appRootDir: SIMPLE_APP, mixinSources: [MIXIN_SOURCES] };
+      var instructions = boot.compile(options);
+      expect(instructions.mixins).to.not.have.property('TimeStamps');
+      expect(instructions.mixins).to.have.property('Other');
+    });
+
+    it('supports mixin name normalization - classify (default)', function() {
+      var options = { appRootDir: SIMPLE_APP };
+      var instructions = boot.compile(options);
+      expect(instructions.mixins).to.have.property('Example');
+      expect(instructions.mixins).to.have.property('Foo');
+      expect(instructions.mixins).to.have.property('TimeStamps');
```

Comment:
Same comment as above. To avoid repetition in these four similar tests you can create a helper to create a set of named mixin files that can be used as `givenMixinFiles('example', 'foo', 'time-stamps')`. Or perhaps `givenAppWithMixinFiles` that would call `appdir.createConfigFilesSync` too.

```js
it('supports mixin name normalization - classify (default)', function() {
  givenAppWithMixinFiles('example', 'foo', 'time-stamps');

  var options = { appRootDir: appdir.PATH };
  var instructions = boot.compile(options);
  // asserts
});
```